src/common/replication_common.cpp

```diff
@@ -348,6 +350,10 @@ void replication_options::initialize()
         "log_private_reserve_max_time_seconds",
         log_private_reserve_max_time_seconds,
         "max time in seconds of useless private log to be reserved");
+    log_private_block_bytes = dsn_config_get_value_int64("replication",
```

Comment:
Now a new gflag like config defination is introduced, you can use it instead.
See https://github.com/XiaoMi/rdsn/blob/552169e0b10000d94113d5715be1ee4a1c58352d/include/dsn/utility/flags.h#L56